rero_ils/modules/imports/api.py

```diff
@@ -484,7 +487,10 @@ def get_id(self, json_data):
         :param id: json document
         :return: id of the record
         """
-        return json_data.get('010__').get('a').strip()
+        id = None
+        if json_data.get('010__'):
+            id = json_data.get('010__').get('a').strip()
+        return id
```

Comment:
1. In python, the default return value for a function is `None`
(see: https://realpython.com/python-return-statement/#:~:text=If%20you%20don%E2%80%99t%20supply%20an%20explicit%20return%20statement%20with%20an%20explicit%20return%20value%2C%20then%20Python%20will%20supply%20an%20implicit%20return%20statement%20using%20None%20as%20a%20return%20value)
so, you can just do :
```python
if json_data.get('010__'):
  return json_data.get('010__').get('a').strip()
```
2. Are you sure that tag `010` always have a subfield `a` ?

Reply:
I had already forgotten about it, thank you